src/ui/tsconfig.json

```diff
@@ -1,15 +1,18 @@
 {
   "compilerOptions": {
     "allowJs": true,
```

Comment:
Do we plan on needing any javascript files? Ideally I'd like to fix it to typescript only